tests/openvino/tools/calibrate.py

```diff
@@ -511,18 +521,18 @@ def map_quantize_with_accuracy_control_parameters(pot_parameters):
 
     supported_parameters.update(
         {
-            "maximal_drop": lambda x, _: {"max_drop": x},
+            "maximal_drop": lambda x: {"max_drop": x},
             "max_iter_num": map_max_iter_num,
             "ranking_subset_size": map_ranking_subset_size,
             "tune_hyperparams": map_tune_hyperparams,
             "convert_to_mixed_preset": map_convert_to_mixed_preset,
+            "drop_type": map_drop_type,
```

Comment:
```suggestion
            "drop_type": max_drop_type,
```

Reply:
this is function https://github.com/openvinotoolkit/nncf/blob/2453f34f8298627940fbc0cb1e532f17535bb260/tests/openvino/tools/calibrate.py#L220